spec/models/gias_school_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe GiasSchool do
+  subject do
+    described_class.new(
+      urn: '100000',
+      name: 'school name',
+      address1: 'the address',
+      town: 'anytown',
+      postcode: 'postcode'
+    )
```

Comment:
Can we use factories and fakers to generate these

Reply:
Sure, although I am not certain we get any value from faking the attributes